src/components/Checkout/DonationItem.vue

```diff
@@ -1,170 +1,498 @@
-						v-kv-track-event="['basket', 'Edit Donation']"
-						@click="enterEditDonation"
-						title="Edit Donation"
+					<div
+						v-show="!editDonation"
+						class="tw-hidden md:tw-block tw-text-right"
 					>
-						{{ formattedAmount }}
-						<kv-material-icon
-							role="img"
-							aria-label="Edit Donation"
+						<button
+							class="donation-amount"
+							data-testid="basket-donation-edit-button-combined"
+							v-kv-track-event="['basket', 'Edit Donation']"
+							@click="enterEditDonation"
 							title="Edit Donation"
-							class="edit-donation tw-text-action
+						>
+							{{ formattedAmount }}
+							<kv-material-icon
+								role="img"
+								aria-label="Edit Donation"
+								title="Edit Donation"
+								class="edit-donation tw-text-action
 								tw-w-3.5
 								tw-h-3.5
 								tw-py-0.5
-								tw-hidden md:tw-inline-block
+								tw-align-bottom
 								"
-							name="pencil"
-							:icon="mdiPencil"
+								name="pencil"
+								:icon="mdiPencil"
+							/>
+						</button>
+					</div>
+					<div v-show="editDonation" class="small-12 columns donation-amount-input-wrapper">
+						<kv-text-input
+							class="donation-amount-input"
+							data-testid="basket-donation-edit-input"
+							name="donation"
+							id="donation"
+							v-model="amount"
+							@blur="validateInput"
+							@keyup.enter.prevent="updateDonation()"
 						/>
-					</button>
+						<kv-button
+							variant="secondary"
+							class="update-donation-inline-button"
+							data-testid="basket-donation-edit-submit"
+							@click="updateDonation()"
+						>
+							Update
+						</kv-button>
+						<button
+							class="show-for-medium remove-wrapper"
+							@click="updateLoanAmount('remove')"
+							data-testid="basket-donation-remove"
+						>
+							<kv-material-icon
+								class="remove-x tw-text-tertiary"
+								name="small-x"
+								:from-sprite="true"
+								title="Remove donation"
+							/>
+						</button>
+					</div>
+					<donate-repayments
+						v-if="hasLoans"
+						@updating-totals="$emit('updating-totals', $event)"
+						@refreshtotals="$emit('refreshtotals')"
+					/>
 				</div>
-
-				<div v-if="hasLoans">
-					<div
-						class="donation-tagline tw-text-small tw-text-secondary tw-my-1"
-						data-testid="basket-donation-tagline"
-						v-html="donationTagLine"
+			</div>
+		</template>
+		<template v-if="donateItemExperimentVersion === 'c'">
+			<div class="tw-flex tw-items-center">
+				<span class="tw-text-secondary tw-text-small tw-pr-1">Donate to Kiva</span>
+				<hr class="tw-flex-grow">
+			</div>
+			<h3>Help our team find more borrowers</h3>
+			<div class="tw-flex tw-flex-col md:tw-flex-row tw-pb-5 tw-mt-2">
+				<!-- donation image -->
+				<div class="tw-hidden md:tw-block tw-flex-none md:tw-mr-3 lg:tw-mr-4.5">
+					<img
+						class="donation-img tw-w-12 lg:tw-w-13 tw-h-12 lg:tw-h-13 tw-rounded"
+						src="@/assets/images/team_donate.png"
```

Comment:
Did we want to do an image require here as well?

Reply:
> Did we want to do an image require here as well?

Is there any benefit to using the image require?

Reply:
It might help with deferring loading, but it was a difference I noticed between the sections.